ci/check/clang_tidy_diff.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python
```

Comment:
I think it would be better if we could download both files from the Oneflow-Inc/llvm-project repo before we do the check, so that we don't need to maintain two copies at the same time